docs/lang/articles/advanced/performance.md

```diff
@@ -66,3 +66,100 @@ def func():
     for i in range(8192):  # no decorator, use default settings
         ...
 ```
+
+# Local Storage Optimizations
+
+Taichi comes with a few optimizations that leverages the *fast memory* (e.g. CUDA shared memory, L1 cache) for performance optimization.
+The idea is straightforward: Where possible, Taichi substitues the accesses to the global memroy (slow) with that to the local one (fast),
```

Comment:
line 73: 
`substitues` >>>` substitutes`
`memroy` >>> `memory` 
and a few typos detected by VS Code Spell Checker ;)
line 103: `exitting` >>> `exiting` 
line 120: `specificially` >>> `specifically`

Reply:
Thanks!